apps/platform/src/components/ExternalLink/CrisprDepmapLink.jsx

```diff
@@ -6,7 +6,7 @@ import Link from '../Link';
 
 const useStyles = makeStyles(theme => ({
   helpIcon: {
-    fontSize: '10px',
+    fontSize: '0.875rem !important',
```

Comment:
Why is not working without the **!important** ?

Reply:
Address !important in different issue https://github.com/opentargets/issues/issues/3062